pkg/auth/handler/mfa/create_oob.go

```diff
@@ -80,17 +66,17 @@ const CreateOOBRequestSchema = `
 			"additionalProperties": false,
 			"properties": {
 				"channel": { "enum": ["sms"] },
-				"phone": { "type": "string" },
-				"authn_session_token": { "type": "string" }
+				"phone": { "type": "string", "format": "phone" },
+				"authn_session_token": { "type": "string", "minLength": 1 }
 			},
 			"required": ["channel", "phone"]
 		},
 		{
 			"additionalProperties": false,
 			"properties": {
 				"channel": { "enum": ["email"] },
-				"email": { "type": "string" },
-				"authn_session_token": { "type": "string" }
+				"email": { "type": "string", "format": "email" },
```

Comment:
I guess the implementation of `format: email` allow name in the address. Can you verify if it is true? If yes, can we introduce our own format that only allow email address without name?  We have `EnsureAddressOnly` implemented already.